### PR TITLE
Dialogflow to gcp

### PR DESCRIPTION
**Pull request by nnegrey**
Moving samples from: https://github.com/dialogflow/dialogflow-java-client-v2

To here.

### CODE REVIEW

File: dialogflow/cloud-client/src/main/java/com/example/dialogflow/ContextManagement.java [hunk 1]
```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright 2018 Google Inc.
```
[the remark this answers is not in the document]
Comment (nnegrey) [reply dialogflow/cloud-client/src/main/java/com/example/dialogflow/ContextManagement.java]: I'm getting a style issue with LLC, is it supposed to be `LLC.`?

[the remark this answers is not in the document]
Comment (kurtisvg) [reply dialogflow/cloud-client/src/main/java/com/example/dialogflow/ContextManagement.java]: No, but you need to update the parent's in the pom.xml so it's using the updated checkstyle (sorry I missed this in the initial review):
```
  <!--
    The parent pom defines common style checks and testing strategies for our samples.
    Removing or replacing it should not affect the execution of the samples in anyway.
  -->
  <parent>
    <groupId>com.google.cloud.samples</groupId>
    <artifactId>shared-configuration</artifactId>
    <version>1.0.9</version>
  </parent>
```

**Comment by nnegrey** [comment]
Not sure what this error is: `Caused by: io.grpc.StatusRuntimeException: INVALID_ARGUMENT: Current user is null` gonna try running again. Haven't seen it before.

**Comment by nnegrey** [comment]
Updated some more region tags. PTAL
Should be good to go now.

**Pull request merged**